apps/full-site-editing/full-site-editing-plugin/donations/donations.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Donations block
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE\Earn\Donations;
+
+/**
+ * Initialize the Donations block.
+ */
+function fse_donations_block() {
+	$asset_file = include plugin_dir_path( __FILE__ ) . 'dist/donations.asset.php';
+
+	wp_enqueue_script(
+		'donations',
+		plugins_url( 'dist/donations.js', __FILE__ ),
+		$asset_file['dependencies'],
+		$asset_file['version'],
+		true
+	);
+}
+
+add_action( 'init', 'A8C\FSE\Earn\Donations\fse_donations_block' );
```

Comment:
does `init` happen before `plugins_loaded`? in that case, it probably never calls fse_donations_block, since the file is only loaded after plugins loaded

Reply:
It was so much sillier than that: I had the `name` for `registerBlockType` as just `donations`, but it needs a prefix to be valid (eg. `a8c/donations`). 🙃

Reply:
😆